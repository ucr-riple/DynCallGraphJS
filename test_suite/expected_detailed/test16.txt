{
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test16.js:1:1:31:5)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test16.js:1:2:31:2)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test16.js:30:3:30:24)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test16.js:3:16:26:6)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test16.js:30:3:30:47)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test16.js:13:16:24:10)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test16.js:15:25:15:63)': [
        'call (Native)(/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test16.js:15:25:15:63)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test16.js:16:40:16:80)': [
        'max (Native)(/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test16.js:16:40:16:80)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test16.js:17:26:17:61)': [
        'Array (Native)(/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test16.js:17:26:17:61)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test16.js:18:28:18:64)': [
        'call (Native)(/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test16.js:18:28:18:64)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test16.js:22:13:22:41)': [
        'concat (Native)(/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test16.js:22:13:22:41)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test16.js:22:13:22:64)': [
        'concat (Native)(/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test16.js:22:13:22:64)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test16.js:20:18:23:12)': [
        'apply (Native)(/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test16.js:20:18:23:12)'
    ],
    'apply (Native) (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test16.js:20:18:23:12)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test16.js:27:19:29:6)'
    ]
}