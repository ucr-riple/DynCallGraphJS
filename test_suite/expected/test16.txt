{ '(test16.js:1:1:31:5)':
   [ '(test16.js:1:2:31:2)' ],
  '(test16.js:30:3:30:24)':
   [ '(test16.js:3:16:26:6)' ],
  '(test16.js:30:3:30:47)':
   [ '(test16.js:13:16:24:10)' ],
  '(test16.js:15:25:15:63)': [ 'call (Native)' ],
  '(test16.js:16:40:16:80)': [ 'max (Native)' ],
  '(test16.js:17:26:17:61)': [ 'Array (Native)' ],
  '(test16.js:18:28:18:64)': [ 'call (Native)' ],
  '(test16.js:22:13:22:41)': [ 'concat (Native)' ],
  '(test16.js:22:13:22:64)': [ 'concat (Native)' ],
  '(test16.js:20:18:23:12)': [ 'apply (Native)' ],
  'apply (Native)':
   [ '(test16.js:27:19:29:6)' ] }